experiment_params/train_config_default.yaml

```
@@ -20,11 +20,12 @@ networks:
     strides: [2, 2, 2]  # first + hidden + last
     out_channels: 16  # Channels of q, and p splitted
   hamiltonian:
-    hidden_conv_layers: 4
+    hidden_conv_layers: 3
     in_shape: [16, 4, 4]  # Should be coherent with transformer output
-    n_filters: [32, 64, 64, 64, 64, 64]  # first + hidden
-    kernel_sizes: [3, 3, 3, 3, 3, 3]  # first + hidden + last
-    strides: [1, 1, 1, 1, 1, 1]  # first + hidden + last
+    n_filters: [32, 64, 64, 64]  # first + hidden
+    kernel_sizes: [3, 2, 2, 2, 2]  # first + hidden + last
+    strides: [1, 2, 1, 1, 1]  # first + hidden + last
```

```suggestion
    strides: [1, 2, 1, 1, 2]  # first + hidden + last
```
last one with stride 2

---

I think stride 2 in the last one doesn't make a lot of sense since the input is 2x2 and the kernel is 2x2